openedx/features/enterprise_support/api.py

```diff
@@ -473,16 +473,18 @@ def consent_needed_for_course(request, user, course_id, enrollment_exists=False)
     if not enterprise_learner_details:
         consent_needed = False
     else:
+        currently_active_enterprise = enterprise_customer_uuid_for_request(request)
         client = ConsentApiClient(user=request.user)
         consent_needed = any(
```

Comment:
Why do we need the 'any' here?

Reply:
We only need one True result. any() will cause the iteration to short-circuited if we find one